infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -708,7 +733,34 @@ public void updateTopology(LogReplicationClusterInfo.TopologyConfigurationMsg to
         return null;
     }
 
+    @Override
+    public void forceSnapshotSync(String clusterId) {
+        if (localClusterDescriptor.getRole() == ClusterRole.STANDBY) {
+            String errorStr = "The forceSnapshotSync command is not supported on standby cluster: " + localNodeDescriptor;
+            log.error(errorStr);
+            throw new RuntimeException(errorStr);
+        }
+
+        log.info("Received the forceSnapshotSync command at node {} and will write to the replicationTable.", localNodeDescriptor);
```

Comment:
We can make this log message shorter for readability, anyways it will only show on the node which received it (no need to specify the localNodeDescriptor). Received forceSnapshotSync